### DISCUSSION TITLE
Disable autofilling of EME form

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#372] [Q&A] [answered]
Hi Franky,

I have an EME form on a post that I am setting as "Private" and using "Temporary Login Without Password" to allow selected people to access it.

After accessing the form, the first & last names and email are all autofilled with this temporary account's credentials. I set "Autocomplete sources" to "None" and the form is still autofilled.

How can I prevent the form from being autofilled?

**Comment by liedekef** [comment, marked as answer]
If using "private" it means the person needs to be logged in, and then the info from the logged in user is always taken. If the form option "Require WP membership for booking" is not set, people can clear the autocompleted info and fill out something else.
If you don't want that: use "Unlisted" and optionally put a password on the WP page itself.

**Reply by refugeindhamma** [reply]
The option "Require WP membership for booking" is not set. I realised that to clear the autofilled info, I need to give the account "edit_others_posts" permission or else there is no option to clear it.

Anyway, to simplify matters, I just set a password on the post.

For future forms, I will install the "Unlist Plugin" to unlist the posts.

Thanks for your reply.

**Reply by liedekef** [reply]
If "Require WP membership for booking" is not set, every logged in user should be able to clear the autofilled info. I'll check that to make sure.
Edit: just tested this and it works as expected.
